server.py

```diff
@@ -67,14 +78,18 @@ def _set_headers(self, page=None, response_code=200, content_type='text/html'):
         self.end_headers()
 
     def do_GET(self):
-        page = self.path[1:] + '.html'
+
+        if 'favicon.ico' in self.path:
```

Comment:
This makes less errors in your server log, but doesn't completely solve the problem. 
You want the explicit 404 error unless you want to supply some favicon.ico.
Not important right now, so you can skip fixing this.

Reply:
fixed

Reply:
Can't see any fixes. Are you sure you've commited that?

Reply:
Now it's all right